libgalois/include/katana/PropertyGraph.h

```diff
@@ -427,7 +423,8 @@ class KATANA_EXPORT PropertyGraph {
         .property_fn = &PropertyGraph::GetEdgeProperty,
         .properties_fn = &PropertyGraph::edge_properties,
         .add_properties_fn = &PropertyGraph::AddEdgeProperties,
-        .remove_property_fn = &PropertyGraph::RemoveEdgeProperty,
+        .remove_property_int = &PropertyGraph::RemoveEdgeProperty,
+        .remove_property_str = &PropertyGraph::RemoveEdgeProperty,
     };
   }
 
```

Comment:
Naive question, is the Edge vs Node property view the primary reason this interface uses function pointers?

Reply:
These aren't so necessary when you are just manipulating an arrow table.  But when you want to write a routine that modifies `Properties`, these allow you to write a single routine that you can use for both node and edge properties.